### ISSUE TITLE
Range Input - WCAG Rule 2.5.7 Dragging Movements

### ISSUE THREAD

**Issue by KennyAtHPE** [grommet/grommet#7599]
Range Input fails this WCAG rule because there are no single-pointer alternatives to control dragging movements.

**Suggested changes**
 Allow users to type a number value into an input or click simple +/- buttons to adjust the slider.
